static/app/views/dashboards/widgetBuilder/buildSteps/filterResultsStep/spansSearchBar.tsx

```diff
@@ -0,0 +1,31 @@
+import {EAPSpanSearchQueryBuilder} from 'sentry/components/performance/spanSearchQueryBuilder';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import type {WidgetBuilderSearchBarProps} from 'sentry/views/dashboards/datasetConfig/base';
+import {useSpanTags} from 'sentry/views/explore/contexts/spanTagsContext';
+
+/**
+ * A search bar for exploring tags and values for spans in Dashboards.
+ * This assumes that the dataset used is the EAP spans dataset.
+ */
+function SpansSearchBar({
+  widgetQuery,
+  onSearch,
+}: Pick<WidgetBuilderSearchBarProps, 'widgetQuery' | 'onSearch'>) {
+  const {
+    selection: {projects},
+  } = usePageFilters();
+  const numberTags = useSpanTags('number');
+  const stringTags = useSpanTags('string');
+  return (
+    <EAPSpanSearchQueryBuilder
+      initialQuery={widgetQuery.conditions}
+      onSearch={onSearch}
+      numberTags={numberTags}
+      stringTags={stringTags}
+      searchSource="explore"
```

Comment:
```suggestion
      searchSource="dashboards"
```

Reply:
Good catch, I actually thought this was the field that mapped to some enum to categorize what kind of search it was so you can return the right type.

Looks like this is just for analytics and there's a prop on the search bar being used (i.e. one of its children) called `recentSearches` and that's the enum